### ISSUE TITLE
Interrupting unattended installation can break it

### ISSUE THREAD

**Issue by mstormi** [openhab/openhabian#952]
I just interrupted an unattended install but on reboot it fails to continue on "apt-get --yes upgrade"
Manual execution of that resulted in "E: dpkg was interrupted, you must manually run 'sudo dpkg --configure -a' to correct the problem."
After I did and rebooted, installation continued.
So while of course a user should not interrupt, and this problem probably is not specific to the new image, some users do like I did, and we should take care of that, i.e run dpkg --configure -a if needed.

**Comment by trumpton** [comment]
I had exactly the same issue - for me, the install just went into a continuous reboot cycle (which of course kept deleting the log file) so I ended up sshing in and killing off the first install process and running it by hand with the full debug option enabled.

From that I could see the offending line was a dpkg install, which, when I ran that line manually, I get the message:

`E: dpkg was interrupted, you must manually run 'sudo dpkg --configure -a' to correct the problem. `

Doing as instructed unblocked the installation for me and got things going again.

Could this --configure command simply be added to the first install script - when I ran it a second time, it just returned without complaint.
